lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -77,6 +76,26 @@ llvm::Error setTensorType(const caffe2::TensorProto &in, Tensor *T) {
     RETURN_ERR("Only float and index tensors are supported");
   }
 }
+
+llvm::Error setTensorType(const caffe2::QTensorProto &in, Tensor *T) {
+  std::vector<size_t> dim;
+  for (auto d : in.dims()) {
+    if (d == 0) {
+      RETURN_ERR("0 dimemsion qtensor is not supported");
+    }
+    dim.push_back(d);
+  }
+
+  if (in.data_type() == caffe2::TensorProto::UINT8) {
+    T->reset(ElemKind::Int8QTy, dim, in.scale(), in.bias() - OFFSETSHIFT);
```

Comment:
what is the use case for what's checked in line 72?
```
else if (in.data_type() == caffe2::TensorProto::UINT8) {
    T->reset(ElemKind::Int8QTy, dim, 1.0, 0);
```

Reply:
@rdzhabarov These are two different branches, in #72 we assume the incoming tensor is a non-quantized tensor, and only use Int8QTy to represent a int8 tensor. Here we know it is a quantized tensor (by knowing protobuf is a QTensorProto), we treated it like real quantized tensor.

Reply:
linked PR seems to be unrelated.
What is the case when tensor is int8 and non quantized?

Reply:
oh it is not the pr I want to link, it is the 72nd line in this file...
I think you have the point, right now glow takes all int8 input as Int8QTy. do we have normal int8ty as well?